### PR TITLE
Update readme for dotnet

### LINKED ISSUE #179: Update README to reflect .NET backend

### PR DESCRIPTION
**Pull request by GodVenn**
Closes #179

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -1,12 +1,38 @@
...
+The backend of flotilla is created using ASP.NET.  
+Useful documentation of concepts and features in the .NET frameworks can be found 
+[here](https://docs.microsoft.com/en-us/dotnet/fundamentals/).
+
+## Setup
+To set up the backend on Windows/Mac, install visual studio and include the "ASP.NET and web development" workload during install.  
+If you already have visual studio installed, you can open the "Visual Studio Installer" and modify your install to add the workload.
+
+To set up the backend on linux..  
+CHRISTIAN PLEASE HELP ME
```
Comment (Christdej) [inline backend/README.md:12]: To set up the backend, install .NET for linux [here](https://docs.microsoft.com/en-us/dotnet/core/install/linux).

File: backend/README.md [hunk 2]
```diff
@@ -37,3 +63,12 @@ cd backend
...
   
+## Configuration  
+
+The project has two [appsettings](https://docs.microsoft.com/en-us/iis-administration/configuration/appsettings.json)
+files.  
+The base `appsettings.json` file is for common variables across all environments, while the
+`appsetings.Development.json` file is for variables specific to the Dev environments, such as the client ID's for the
+various app registrations used in development.
+
+In the future, most of the config should probably be pulled from various Azure Key Vaults, one for each environment.
```
Comment (Christdej) [inline backend/README.md:74]: Unnecessary sentence? Rather make an issue than adding TODOs in README

File: backend/README.md [hunk 3]
```diff
@@ -1,12 +1,38 @@
...
+```
+  "AzureAd": {
+    "ClientSecret": "ThisIsASecret"
+  }
+```
+
 ## Run
 
-To build and run the app, run the following command in the backend folder:
+To build and run the app, run the following command in the backend/api folder:
```
Comment (Christdej) [inline backend/README.md:26]: We could consider writing:
```suggestion
To build and run the app, run the following command 'dotnet run --project api':
```
To have all the commands from the same folder. Thoughts?

**Review by Christdej — CHANGES_REQUESTED** [review]

File: backend/README.md (hunk 2 above)
Comment (GodVenn) [reply backend/README.md:74]: Agreed

File: backend/README.md (hunk 3 above)
Comment (GodVenn) [reply backend/README.md:26]: Sure!

**Review by Christdej — APPROVED** [review]

**Pull request merged**